packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.Props.ts

```diff
@@ -174,9 +174,17 @@ export interface IGroup {
 
 export interface IGroupRenderProps {
 
-  /** Boolean indicating if all groups are in collapsed state. */
+  /**
+   * Boolean indicating if all groups are in collapsed state.
+   * @deprecated
+   */
   isAllGroupsCollapsed?: boolean;
 
+  /**
+   * Boolean indicating if all groups are initially in collapsed state.
+   */
+  defaultAllGroupsCollapsed?: boolean;
```

Comment:
can you call it `initial..` to be more consistent?

See https://github.com/OfficeDev/office-ui-fabric-react/search?p=5&q=default%2A+filename%3A%2A.Props.ts&type=&utf8=%E2%9C%93 vs https://github.com/OfficeDev/office-ui-fabric-react/search?utf8=%E2%9C%93&q=initial*+filename%3A*.Props.ts&type=

Reply:
sure

Reply:
done!